addons/web/static/src/legacy/scss/fields.scss

```diff
@@ -51,6 +51,16 @@
     }
 }
 
+// Invalid background
+// Avoid overlapping the bg in the input inside the list view nor on
+// the order_line because the o_invalid_cell already has the background
+:not(.o_invalid_cell) {
```

Comment:
`o_invalid_cell` it's a `listView` class; rules affecting it's behaviour it should be located there ;)